website/static/js/registrationUtils.js

```diff
@@ -260,10 +260,15 @@ var MetaSchema = function(params) {
     self.fulfills = params.fulfills || [];
     self.messages = params.messages || {};
 
+    var count = 0;
     $.each(self.schema.pages, function(i, page) {
         var mapped = {};
         $.each(page.questions, function(qid, question) {
-            mapped[qid] = new Question(question, qid);
+            // This is necessary now since page.questions is a list
```

Comment:
The first argument to the callback in `$.each(array, callback)` is an index. I suspect you can use that instead of tracking (and incrementing) `count` separately. http://api.jquery.com/jquery.each/